Multisig/UI/Settings/OwnerKeyManagement/RequestToAddOwner/ValidateRequestToAddOwnerViewController.swift

```diff
@@ -98,11 +98,13 @@ class ValidateRequestToAddOwnerViewController: UIViewController {
 
     func handleSafeNotAdded(info: SCGModels.SafeInfoExtended) {
         // safe must be added to the app to continue
+        Tracker.trackEvent(.screenOwnerFromLinkNoSafe)
         let noSafeVC = AddOwnerExceptionViewController.safeNotFound(
             address: parameters.safeAddress,
             chain: parameters.chain,
             onAdd: { [unowned self] in
-                // add safe
+                Tracker.trackEvent(.userOwnerFromLinkSafeNameAdded,
```

Comment:
@DmitryBespalov @sche are you ok with using this event here or shall I rename it

Reply:
@biafra23 See here

Reply:
ok